src/state/models/discovery/feeds.ts

```diff
@@ -48,16 +54,34 @@ export class FeedsDiscoveryModel {
     this._xLoading()
     try {
       const res =
-        await this.rootStore.agent.app.bsky.unspecced.getPopularFeedGenerators(
-          {},
-        )
+        await this.rootStore.agent.app.bsky.unspecced.getPopularFeedGenerators({
+          limit: DEFAULT_LIMIT,
+        })
       this._replaceAll(res)
       this._xIdle()
     } catch (e: any) {
       this._xIdle(e)
     }
   })
 
+  loadMore = bundleAsync(async () => {
+    this._xLoading()
+    if (!this.hasMore) {
+      return
+    }
```

Comment:
Should the `this._xLoading()` call happen after the `this.hasMore` check?

Reply:
You're right. Good catch!